security/cmd/node_agent/main.go

```diff
@@ -28,6 +28,10 @@ import (
 	nvm "istio.io/istio/security/pkg/nodeagent/vm"
 )
 
+var (
+	GoogleCAAddr = "istioca.googleapis.com:443"
```

Comment:
Aren't those referenced somewhere else? Let's reuse the same var, refactoring if needed.

Reply:
I don't think so. Searching the string "istioca.googleapis.com:443" doesn't give back any result, except in one yaml file.

Reply:
Then use env var as well instead of hard coding.

@quanjielin I think we recently switched from flag to env var. Can you confirm?

Reply:
yes, ```CAEndpoint``` is passed from env today(https://github.com/istio/istio/blob/master/security/cmd/node_agent_k8s/main.go#L303)

Reply:
Thanks @quanjielin, why did we switch from using flags to env vars, just curious?

Reply:
Just want to confirm, in `master`, Node Agent is not using env, but it is in `release-1.1` (https://github.com/istio/istio/blob/release-1.1/security/cmd/node_agent_k8s/main.go). Is it safe to say that `release-1.1` is more up to date?